### PR TITLE
chore: rename components to "pds"

### PR DESCRIPTION
**Pull request by ju-Skinner**
# Description

This is a PR that will rename the components and processes to use the new Design System name. The currently agreed naming is as follows:

Component prefix: `pds` e.g `pds-icon`
Component Class Name: `Pds[ComponentName]` e.g `PdsIcon`
Tokens prefix: `--pine` e.g `--pine-background-color`

### CODE REVIEW

File: libs/core/src/components/pds-icon/pds-icon.docs.stories.mdx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-  extractedArgs.name.options = Object.values(sageIconsJson["icons"]).map((icon) => (icon.name));
+  extractedArgs.name.options = Object.values(pdsIconsJson["icons"]).map((icon) => (icon.name));
   return extractedArgs;
 }
 
-<Meta title="components/Icon" component="sage-icon" argTypes={argTypes()}/>
+<Meta title="components/Icon" component="pds-icon" argTypes={argTypes()}/>
 
 # Icon
 Displays Icons used with the Sage Design System
```
Comment (pixelflips) [inline libs/core/src/components/pds-icon/pds-icon.docs.stories.mdx]: Should Sage be updated here?

File: libs/icons/src/index.html [hunk 2]
```diff
@@ -5,28 +5,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
     <title>Stencil Component Starter</title>
 
-    <script type="module" src="/build/sage-icons.esm.js"></script>
-    <script nomodule src="/build/sage-icons.js"></script>
+    <script type="module" src="/build/pds-icons.esm.js"></script>
+    <script nomodule src="/build/pds-icons.js"></script>
   </head>
   <body>
     <h1>Sage Icons</h1>
```
Comment (pixelflips) [inline libs/icons/src/index.html]: Should Sage be updated here?

**Review by pixelflips — APPROVED** [review]
A couple of non-blocking comments, but overall LGTM! 👍🏼

File: libs/core/src/components/pds-icon/pds-icon.docs.stories.mdx (hunk 1 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-icon/pds-icon.docs.stories.mdx]: Good catch. Thank you,

File: libs/icons/src/index.html (hunk 2 above)
Comment (ju-Skinner) [reply libs/icons/src/index.html]: Another good  catch.

**Review by QuintonJason — APPROVED** [review]
Good to go!

**Review by anechol — APPROVED** [review]

**Review by pixelflips — APPROVED** [review]
Checked the latest round of updates and all seem fine and working as expected. Others broke due to token name changes, but that's to be expected. Great work! 👍🏼

**Review by QuintonJason — APPROVED** [review]
Everything up to input looks good

**Review by teenwolfblitzer — APPROVED** [review]
😎 LGTM. Safe to assume the static elements (branding, "get support" page) will be updated separately?

**Comment by ju-Skinner** [comment]
> 😎 LGTM. Safe to assume the static elements (branding, "get support" page) will be updated separately?

Yes, you are correct @monkeypox8. Things like

* CDN / Brand URLs
* Images

**Review by QuintonJason — APPROVED** [review]
Good stuff

File: libs/core/src/components/pds-textarea/pds-textarea.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  font-size: var(--label-font-size);
+  font-weight: var(--label-font-weight);
+  line-height: var(--label-line-height);
+  margin-block-end: var(--label-margin-block-end);
+}
+
+.pds-textarea__field {
+  background-color: var(--background-color);
+  border: var(pine-border-interactive);
+  border-radius: var(pine-border-radius-md);
```
Comment (pixelflips) [inline libs/core/src/components/pds-textarea/pds-textarea.scss]: These are missing the `--` in the variable names

**Review by pixelflips — CHANGES_REQUESTED** [review]
Found a minor styling issue, otherwise looks good.

Not sure if an update is needed now, but the `libs/core/src/stories/Support.stories.mdx` file also mentions Sage.

**Comment by ju-Skinner** [comment]
> Found a minor styling issue, otherwise looks good.
> 
> Not sure if an update is needed now, but the `libs/core/src/stories/Support.stories.mdx` file also mentions Sage.

I fixed the story...You can see the change in the `fixup` commit. I'll squash it once everyone gets their review complete.

**Review by pixelflips — APPROVED** [review]
LGTM! Nice work!

**Review by anechol — APPROVED** [review]
LGTM

**Review by teenwolfblitzer — APPROVED** [review]
🌲

**Pull request merged**